docker.env.sample

```diff
@@ -0,0 +1,3 @@
+NODE_ENV=development
```

Comment:
I wouldn't like to have a separate place where we need to maintain all env variables. There's already .env.sample. This exact thing has caused ~ 50% of all issues we've had with Docker in our current project (not remembering to update envs to Docker setup). 

Would there be a better way to get env vars inside docker?